app/ui/src/app/common/empty-state-card/index.ts

```diff
@@ -0,0 +1,22 @@
+import { Component } from '@angular/core';
+
+@Component({
+  selector: 'syndesis-empty-state-card',
+  template: `
+  <div class="card empty-state-card">
+    <div class="card-pf card-pf-view card-pf-view-select card-pf-view-single-select">
+      <div class="card-pf-body">
+        <div class="card-pf-top-element">
+          <ng-content select=".empty-state-card__header"></ng-content>
+        </div>
+        <div class="card-pf-info text-center">
+          <ng-content select=".empty-state-card__body"></ng-content>
+        </div>
+      </div>
+    </div>
+  </div>
+  `,
+  styleUrls: ['empty-state-card.component.scss']
+})
+
+export class EmptyStateCardComponent {}
```

Comment:
I'm afraid you'll have to rename this file to `empty-state-card.component.ts` instead. The Angular convention is to name files by matching its class content in kebab-case.

We use `index.ts`  files to abstract the contents of folders to its consumers. This practice is better known as "barrel": https://angular.io/guide/glossary#barrel

Reply:
Got it! Updated the branch.